src/main/java/org/tikv/common/log/SlowLogImpl.java

```diff
@@ -43,19 +43,13 @@
 
   private final long slowThresholdMS;
 
-  /** Key-Value pairs which will be logged, e.g. function name, key, region, etc. */
-  private final Map<String, String> properties;
+  private final long traceId;
 
-  public SlowLogImpl(long slowThresholdMS, Map<String, String> properties) {
+  public SlowLogImpl(long slowThresholdMS) {
     this.startMS = System.currentTimeMillis();
     this.startNS = System.nanoTime();
     this.slowThresholdMS = slowThresholdMS;
-    this.properties = new HashMap<>(properties);
-  }
-
-  @Override
-  public void addProperty(String key, String value) {
-    this.properties.put(key, value);
+    traceId = random.nextLong();
```

Comment:
why use random trace id? it could be conflicted with other traces in other requests, is it acceptable, why?

Reply:
OpenTelemetry also use random number as trace_id (See the [doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md)). It is widely considered acceptable to rely on random numbers being unique if those numbers have enough bits. See https://softwareengineering.stackexchange.com/a/339140.

A 128-bit UUID will be better than current 64-bit random number (and is used by OpenTelemetry), but uint64 as trace_id has already used in [kvproto](https://github.com/pingcap/kvproto/commit/3fa8fa04f898c8b2f5f9b7b8576f38016cb2d5e6#diff-a546353419478c2fd5b495073d1eee706048b7524e9973f0aa846e0959c5ba42), so I just follow that.